omnipaxos_core/Cargo.toml

```
@@ -26,10 +26,9 @@ rand = "0.8.4"
 [features]
 batch_accept = []
 continued_leader_reconfiguration = []
-logging  = [ "slog", "slog-term", "slog-async"]
-hocon_config = [ "hocon" ]
-
-default = ["continued_leader_reconfiguration", "batch_accept", "hocon_config"]
+logging  = ["slog", "slog-term", "slog-async"]
+toml-config = ["serde", "toml"]
```

Let's be consistent with the other features in the library and use underscore.

```suggestion
toml_config = ["serde", "toml"]
```